src/components/Modal/Modal.spec.js

```diff
@@ -61,12 +65,12 @@ describe('Modal', () => {
 
   it('should open', () => {
     const wrapper = openModal(defaultModal);
-    expect(wrapper.find('Card')).toHaveLength(1);
+    expect(wrapper.find('[data-testid="card"]')).toHaveLength(1);
   });
 
   it('should close', async () => {
     const wrapper = openModal(defaultModal);
-    const closeButton = wrapper.find('SvgButton').find('button');
+    const closeButton = wrapper.find('[data-testid="close"]').find('button');
```

Comment:
I don't think this PR is the right place to introduce a different convention for these ids :/ We have been using `data-selector` elsewhere.

Reply:
This is something that is used only in the tests. It is not a convention for adding data selectors in the components themselves. It is not the same thing.

Reply:
We are also using the selectors in the tests though. At the moment, the responsibility of test selectors and external selectors is handled by the same data attribute.

My personal opinion was to separate the two but still, this doesn't follow the convention we're using elsewhere.